invenio_rdm_records/models.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+# Copyright (C) 2019 Northwestern University,
+#                    Galter Health Sciences Library & Learning Center.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Models for Invenio RDM Records."""
+
+import json
+from os.path import dirname, join
+
+from elasticsearch_dsl.utils import AttrDict
+from jsonref import JsonRef
+
+
+class AccessRight(object):
```

Comment:
can be done later: This seems like a Jinja2 filter because it is really all about the frontend. Could be moved to `views.py` and adapted.

Reply:
Created a TODO :+1:

Reply:
Added a TODO